cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/model/ClusterModel.java

```diff
@@ -1075,15 +1082,11 @@ private void addSingleBrokerStats(String host, int id, Broker.State state, doubl
       return _brokerStats;
     }
 
-    /**
-     * Return a valid JSON encoded string
-     *
-     * @param version JSON version
-     */
-    public String getJSONString(int version) {
+    @Override
+    public String getJSONString(CruiseControlParameters parameters) {
       Gson gson = new Gson();
       Map<String, Object> jsonStructure = getJsonStructure();
-      jsonStructure.put(VERSION, version);
+      jsonStructure.put(VERSION, JSON_VERSION);
```

Comment:
`BrokerStats` is also one field of  `KafkaOptimizationResult` and in there we already have JSON version in `KafkaOptimizationResult`. looks like in that case the version field is redundant.